libr/include/r_util/r_str.h

```diff
@@ -21,6 +21,12 @@ typedef enum {
 
 typedef int (*RStrRangeCallback) (void *, int);
 
+typedef struct { ut8 ch; ut8 hx; } charset;
+
+typedef struct r_charset_t {
+	charset custom_charset[256];
```

Comment:
save the dictionary size

Reply:
@trufae yes I have read the pad. I was going to do it.

Reply:
@trufae I have replaced all by a search tree. I am going to show you that.